gh_actions_exporter/metrics.py

```diff
@@ -117,24 +113,11 @@ def runner_type(self, webhook: WebHook) -> str:
             return "self-hosted"
         return "github-hosted"
 
-    def relabel_job_labels(
-        self, relabel: Relabel, labels: List[str]
-    ) -> Dict[str, str or None]:
-        result = {relabel.label: relabel.default}
-        for label in relabel.values:
-            if label in labels:
-                result[relabel.label] = label
-        return result
-
-    def relabel_job_names(self, relabel: Relabel, job: WorkflowJob) -> dict:
-        result = {relabel.label: relabel.default}
-        if job.status == "queued":
-            result[relabel.label] = ""
-        else:
-            for label in relabel.values:
-                if label in job.runner_name:
-                    result[relabel.label] = label
-        return result
+    def flavor_type(self, webhook: WebHook) -> str:
+        for label in webhook.workflow_job.labels:
+            if label in self.settings.job_costs:
+                return label
+        return "flavor"
```

Comment:
Why returning `flavor` when nothing is found?

```suggestion
    def flavor_type(self, webhook: WebHook) -> str or None:
        for label in webhook.workflow_job.labels:
            if label in self.settings.job_costs:
                return label
        return None
```